modules/mono/glue/GodotSharp/GodotSharp/Core/Mathf.cs

```diff
@@ -885,6 +885,28 @@ public static double Exp(double s)
             return Math.Exp(s);
         }
 
+        /// <summary>
+        /// The natural exponential minus one.
+        /// </summary>
+        /// <param name="s">The exponent to raise <c>e</c> to.</param>
+        /// <returns><c>e</c> raised to the power of <paramref name="s"/>, minus one.</returns>
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public static float ExpM1(float s)
```

Comment:
This method is not exposed to scripting, so I would make it `private` if we really need it. Also, it would have to be moved to `MathfEx.cs`.

Reply:
Methods moved to `MathfEx.cs`. I can't use `private` on them though.

Reply:
Oh, I didn't realize this was used outside of `Mathf`, then let's make it `internal`.